src/analysis.js

```diff
@@ -58,9 +58,12 @@ async function requestStack(provider, manifest, url, html = false, opts = {}) {
 		body: provided.content
 	}, opts);
 
-	let resp = await fetch(`${url}/api/v4/analysis`, fetchOptions)
+	const finalUrl = new URL(`${url}/api/v4/analysis`);
+	finalUrl.searchParams.append('recommend', opts['EXHORT_RECOMMENDATIONS_ENABLED'] === 'true' ? 'true' : 'false');
```

Comment:
let's set it only to false and omit it in any other case. I prefer to keep the requests simple.